public/components/SponsorshipEdit/SponsorLogo.react.js

```diff
@@ -60,7 +60,7 @@ export default class SponsorEdit extends React.Component {
     const imageAsset = this.props.logo.assets[0];
 
     return (
-      <div className="tag-edit__field">
+      <div className="tag-edit__field tag-edit__field-logo">
```

Comment:
We're using a BEM naming convention (ideally) in this project, so generally there should be a single class per thing... so as you need a new variant of the `tag-edit__field` class then you can create `tag-edit__field--logo` and then in the SASS do something similar to `%tag-edit__image__add`